docs/v1/P4Runtime-Spec.mdk

```diff
@@ -5394,13 +5408,22 @@ it populates the `MasterArbitrationUpdate` message using the `device_id`,
 also populates the `status` field in the `MasterArbitrationUpdate` according to
 the rules in an [earlier section](#sec-mastership-updates).
 
-The sender need not specify an `election_id`. If the `election_id` is not
-specified, the sender's `election_id` is considered lower than any
+### Unset Election ID
+
+The sender need not specify an `election_id`. If the `election_id` is
+unset, the sender's `election_id` is considered lower than any
```

Comment:
Just out of curiosity, if two controllers have unset election ids, does that mean they are both standbys, and nether can take control?

Reply:
Correct, you can have any number of standbys. A standby can become primary by sending another election request with a higher election ID.